backend/controller/ml_model/shared.py

```diff
@@ -37,6 +37,7 @@ class Timespan:
 
   _training: int
   _predictive: int
+  _exclusion: int
   _consider_datetime: bool
 
   def __init__(self, timespans: list[dict[str, Any]], consider_datetime: bool = False) -> None:
```

Comment:
nit:  I would add pydoc explaining how the args are used to fill in the 4 class attributes (ie, _training).  I'm assuming that's what the constructor is doing, but a pydoc would make that clear.

Reply:
Will add this in the next PR. Thanks!